version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.1-SNAPSHOT"
+version in ThisBuild := "0.0.2-SNAPSHOT"
```

Comment:
Right now there is no official release of `compendium`. IMHO, I don't think it is necessary to bump the version right now. Once `0.0.1` is released, `sbt-org-policies` will set the version to `0.0.2-SNAPSHOT` as its next action.

I think `version` should only be changed to stable versions so the CI makes a release. But it is just my opinion, of course :smile:

Reply:
Ok, sounds good. I will downgrade back to to 0.0.1-SNAPSHOT.